azure-kusto-data/tests/e2e.py

```diff
@@ -0,0 +1,278 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License
+
+import asyncio
+import json
+import os
+import platform
+from datetime import datetime
+from typing import Optional, ClassVar
+
+import pytest
+
+from azure.kusto.data import KustoClient, KustoConnectionStringBuilder
+from azure.kusto.data._models import WellKnownDataSet
+from azure.kusto.data.aio import KustoClient as AsyncKustoClient
+from azure.kusto.data.streaming_response import FrameType
+
+
+@pytest.fixture(params=["ManagedStreaming", "NormalClient"])
+def is_managed_streaming(request):
+    return request.param == "ManagedStreaming"
+
+
+class TestE2E:
```

Comment:
I think the cloud info tests at the end of the ingest e2e can also go here

Reply:
sounds good